css/components/main.css

```diff
@@ -1,26 +1,30 @@
 main {
-    overflow: auto;
-    padding: 30px;
-    display: grid;
-    grid-gap: 24px;
-  }
+  overflow: auto;
+  padding: 30px;
+  display: grid;
+  grid-gap: 24px;
+}
 
-  .profile__funny__icon {
-      text-align: center;
-      height: 200px;
-      border: 3px solid rgb(107, 70, 96);
-      padding: 0.9%;
-      border-radius: 20%;
-  }
+.main.site--hidden {
```

Comment:
`site--hidden` is not quite correct BEM as it is a modifier class from the `main` class. `main--hidden` would be correct here.

Reply:
Ah, but I see you are using it for the `header` as well. Maybe then a utility-class would be best.
Would recommend adding a new `_utility.css` file (not in the components folder) and add a class `hidden` to it which has `display: none !important;` in it. Does that make sense?

Reply:
I do not understand the thing with the utility.css ... I know that site--hidden is not correct BEM, but I needed a class name for main part and the header part as well. I did not want to dobble it as it is the same function. Do you have an example for the utility-thing?

Reply:
I have worked on your feedback in a new branch to be sure it works and I do not ruin my current version. The Branch ist called "branchfeedback" and I posted the Pull Request in Slack